test/arithmetic/fixed_to_float_test.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// fixed_to_float_test.dart
+// Test fixed point to floating point converters.
+//
+// 2024 October 24
+// Author: Soner Yaldiz <soner.yaldiz@intel.com>
+
+import 'dart:io';
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+
+void main() async {
```

Comment:
maybe some more tests to cover corner cases? sweep exhaustive for some small low precision if we can?  normals/subnormals etc?

Reply:
Please see `'Q4.4 to E3M2' below. Exhaustive test from '0 to '1.

Reply:
oh you're right, my bad, nice!